src/routes/customer-resource/customer-resource-show.js

```diff
@@ -14,31 +14,42 @@ class CustomerResourceShowRoute extends Component {
         vendorId: PropTypes.string.isRequired
       }).isRequired
     }).isRequired,
-    model: PropTypes.object.isRequired,
-    requestResource: PropTypes.func.isRequired,
-    toggleSelected: PropTypes.func.isRequired
+    customerResource: PropTypes.object.isRequired,
+    toggleRequest: PropTypes.object.isRequired,
+    makeRequest: PropTypes.func.isRequired
   };
 
   componentWillMount() {
     let { vendorId, packageId, titleId } = this.props.match.params;
-    this.props.requestResource(vendorId, packageId, titleId);
+    this.props.makeRequest('customer-resource', { vendorId, packageId, titleId });
   }
 
+  toggleSelected = () => {
+    let { vendorId, packageId, titleId } = this.props.match.params;
+    let { isSelected } = this.props.customerResource.content;
+
+    this.props.makeRequest('customer-resource-toggle-selected', {
```

Comment:
I go back and forth on these. It feels like the strings such as `customer-resource-toggle-selected'` belongs to the ducks. Have you thought about also `createRequestActions` also to sit along side `createRequestEpic` and `createRequestReducer`?

Reply:
I did think about that. Should that be a helper, or maybe just a simple:

```js
const toggleSelected = makeRequest.bind(null, 'customer-resource-toggle-selected');
```

Reply:
Yeah, I like that, althoug the bind syntax is a tad esoteric, we can just use a function literal.